plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -161,6 +161,19 @@ public DockerConnector(
   }
 
   /**
+   * Gets the API endpoint URL to be used if the `che.workspace.che_server_endpoint` property is
+   * null.
+   *
+   * @return The default API endpoint URL for this connector which is null because docker connector
+   *     uses the default value added in 'che.properties'.
+   * @throws IOException when a problem occurs with docker api calls
+   */
+  @Nullable
+  public String getApiEndpoint() {
```

Comment:
Can you elaborate on how this relates to other methods of DockerConnector

Reply:
see the comment of commit 2991c3eb64bc39f5a11428ffd06ed2a56c79349e for detailed explanations